packages/widgets/src/components/bubble/bubble-chart.widget.tsx

```diff
@@ -4,9 +4,14 @@ import {
   BubbleChartSettings,
   ResponsiveWrapper,
   BubbleLegend,
+  SeriesLegend,
   theme as defaultTheme,
 } from '@keen.io/charts';
 
+import { scaleOrdinal } from 'd3-scale';
```

Comment:
The `keen.io/widgets` do not have `d3-scale` dependency. Please move it to `keen.io/charts` and export helper function like `createColorScale`.